File: Data Structure/Lab 3/MaxMin.c
// Write a program to find position of the smallest number & the largest number 
//from given n numbers.

#include<stdio.h>
void main(){

    int n,a[n],i = 0,j = 0;
    int max = a[0],maxindex = 0;
    int min = a[0],minindex = 0;

    printf("Enter size of an array(n) :");
    scanf("%d",&n);

    for(i = 0; i < n ; i++){
        printf("Enter element :");
        scanf("%d",&a[i]);
    }

    for(i = 0; i < n;i++){
        if(max < a[i]){
            max = a[i];
            maxindex = i;
        }
    }

    for(j = 0; j < n;j++){
        if(min > a[j]){
            min = a[j];
            minindex = j;
        }
    }

    printf("Index of largest element is on : %d\n",maxindex);
    printf("Index of smallest element is on : %d\n",minindex);
}

File: Data Structure(Java)/Lab 2/MaxMin.java
// Write a program to find position of the smallest number & the largest number 
//from given n numbers.

import java.util.Scanner;

public class MaxMin {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter size");
        int n = sc.nextInt();
        int[] arr = new int[n];

        for(int i = 0 ; i < n;i++){
            System.out.print("Enter element at "+i);
            arr[i] = sc.nextInt();
        }

        int max = arr[0];
        int min = arr[0];

        for(int i = 0; i < n;i++){
            if(max < arr[i]){
                max = arr[i];
            }
        }

        for(int i = 0; i < n;i++){
            if(min > arr[i]){
                min = arr[i];
            }
        }

        System.out.println(max);
        System.out.println(min);

        sc.close();
    }
}
